examples/textarea/test.ts

```diff
@@ -0,0 +1,10 @@
+import { q } from "@ariakit/test";
+
+test("markup", () => {
+  expect(q.textbox()).toMatchInlineSnapshot(`
+    <textarea
+      class="textarea"
+      placeholder="Write your comment, be kind"
+    />
+  `);
+});
```

Comment:
It could test some focusable features, like if `data-focus-visible` is added when the user focuses on the element.

Reply:
test added in https://github.com/ariakit/ariakit/pull/4032/commits/1d2def9c5a73b695045be7a593c033c7f852d38a

let me know if you think it needs any more tests